test-runner.js

```
@@ -0,0 +1,173 @@
+        
+      case 'token':
+        console.log('Running handleTokenInfo');
+        await githubController.handleTokenInfo({
+          respond: mockRespond
+        });
+        break;
+        
+      case 'raw':
+        // Directly run generateContributionReport for raw performance testing
+        console.time('Raw report generation');
+        const report = await githubController.generateContributionReport();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)